ffi/diplomat/Cargo.toml

```diff
@@ -36,10 +36,13 @@ denylist = ["bench", "smaller_static"]
 
 # Please keep the features list in sync with the icu_capi_staticlib crate
 [features]
-default = ["provider_fs", "provider_static"]
+default = ["provider_fs", "provider_static", "deserialize_postcard_07", "deserialize_json"]
 provider_fs = ["icu_provider_fs"]
 provider_static = ["icu_testdata"]
 smaller_static = ["provider_static"]
+deserialize_json = ["icu_provider/deserialize_json"]
```

Comment:
Question: To what extend do we want to duplicate these features in other crates? Because one can write `cargo build --feature icu_provider/deserialize_bincode_1` and get the feature enabled

Reply:
In my experience that doesn't always work when building a specific crate.

In the past it didn't work at all (you could only affect features of the current crate, OR you had to build the workspace root) but I think now it sometimes works. Better to abstract away internals IMO.